gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Fri Sep 09 23:28:22 CEST 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-bin.zip
```

Comment:
I downloaded this file and noticed that `gradle/wrapper/gradle-wrapper.jar` files differ. Why? Other than that LGTM!